IDE/STM32Cube/default_conf.ftl

```diff
@@ -534,6 +534,16 @@ extern ${variable.value} ${variable.name};
     #define NO_MD5
 #endif
 
+/* ------------------------------------------------------------------------- */
+/* Crypto Accelaration */
+/* ------------------------------------------------------------------------- */
+#if defined(WOLF_CONF_ARMASM) && WOLF_CONF_ARMASM == 1
+    #define WOLFSSL_ARMASM
```

Comment:
Add comment: "This enabled inline assembly speedups for SHA2, SHA3, AES, ChaCha20/Poly1305 and Ed/Curve25519. These settings work for Cortex M4/M7 and are located in `wolfcrypt/src/port/arm/`."